### ISSUE TITLE
error querying ingesters in Querier.SearchTags: failed to execute f() for [::]:9095: rpc error: code = Unimplemented desc = unknown service tempopb.Querier

### ISSUE THREAD

**Issue by Vinaum8** [grafana/tempo#3981]
**Describe the bug**
The entire deployment is configured, the distributor is returning 1 ingester in distributor:3100/ingester/ring but the query or span ingestion is returning an "unknown service" error.

**To Reproduce**
1. Helm install and connection with MinIO. 
2. curl -u "user:password" -G -s https://gateway-tempo.domain_name/api/v2/search/tags

**Environment:**
 - Infrastructure: EKS Kubernetes
 - Networking: DualStack
 - Deployment tool: ArgoCD - Helm tempo-distributed version 1.17.0

**Additional Context**

**Comment by Vinaum8** [comment]
```
compactor: {"caller":"main.go:118","level":"info","msg":"Starting Tempo","ts":"2024-08-20T16:27:31.551479685Z","version":"(**version=2.5.0**, branch=HEAD, revision=46dad34
distributor: {"caller":"main.go:118","level":"info","msg":"Starting Tempo","ts":"2024-08-20T16:27:33.013798025Z","version":"(**version=2.5.0**, branch=HEAD, revision=46dad34
ingester: {"caller":"main.go:118","level":"info","msg":"Starting Tempo","ts":"2024-08-20T16:27:42.347586918Z","version":"(**version=2.5.0**, branch=HEAD, revision=46dad34
querier: {"caller":"main.go:118","level":"info","msg":"Starting Tempo","ts":"2024-08-20T16:27:41.894605812Z","version":"(**version=2.5.0**, branch=HEAD, revision=46dad34
query frontend: {"caller":"main.go:118","level":"info","msg":"Starting Tempo","ts":"2024-08-20T16:27:43.171309417Z","version":"(**version=2.5.0**, branch=HEAD, revision=46dad34
```

Same versions to all services: 2.5.0

![image](https://github.com/user-attachments/assets/3fae108e-9872-4e0b-858c-5e9d809a5016)

i added the default RF3 as you suggested.

But, my curl: `curl -u "user:password" -G -s https://gateway-tempo.domain_name/api/v2/search/tags`
Return:
`
error querying ingesters in Querier.SearchTags: failed to execute f() for [::]:9095: rpc error: code = Unimplemented desc = unknown service tempopb.Querier`

I have a grafana-loki stack on the same cluster and I compared the grafana-loki distributor with grafana-tempo distributor.

Grafana-tempo:
![image](https://github.com/user-attachments/assets/5901bd4e-0140-437c-8a29-1fe7cf75b7f0)

Grafana-Loki:
![image](https://github.com/user-attachments/assets/159b929b-01b2-4a0d-a326-e16158893d42)

I'll look into it.

**Comment by Vinaum8** [comment]
Well, I managed to solve it!!!

I made the following change to my ingester configuration:
![image](https://github.com/user-attachments/assets/67b8272e-fd6f-4d06-9208-5a020f864938)

However, the documentation suggests that I add this parameter. Therefore, I will request an edit to the [documentation](https://grafana.com/docs/tempo/latest/configuration/network/ipv6/) to inform that only the "enable_inet6: true" property is enough to run with ipv6.

After removing the " address: '::' ", the addresses appeared and the curl command worked.

![image](https://github.com/user-attachments/assets/b04ebf32-e17f-4ef8-b606-07e71ad38426)

Thanks for the initial question @joe-elliott
